packages/components/src/components/livechat-link/livechat-link.scss

```diff
@@ -0,0 +1,14 @@
+.livechat {
+    &__link {
+        background: transparent;
+        border: none;
+        cursor: pointer;
+        margin: 0;
+        padding: 0;
+        color: var(--brand-red-coral);
```

Comment:
color has been given to Text component. Do we still need the SCSS style?

Reply:
I think because of some overlapping, it didn’t work. it works with both styles